src/Containers/Microsoft.NET.Build.Containers/Registry/DefaultManifestOperations.cs

```diff
@@ -42,7 +42,7 @@ public async Task PutAsync(string repositoryName, string reference, ManifestV2 m
     {
         string jsonString = JsonSerializer.SerializeToNode(manifest)?.ToJsonString() ?? "";
         HttpContent manifestUploadContent = new StringContent(jsonString);
-        manifestUploadContent.Headers.ContentType = new MediaTypeHeaderValue(manifest.MediaType);
+        manifestUploadContent.Headers.ContentType = new MediaTypeHeaderValue(manifest.MediaType!);
```

Comment:
same - this may be null some times so we need to have a reasonable default here - in my PR we chose the old docker schema for compatibility, I would suggest doing that here too.

Reply:
yeah, we could do this here
https://github.com/dotnet/sdk/pull/40776/files#diff-7024c1524ae5fbe239cecd57eb4c77dedbaef7d84d93a9c5cd02bec7548a9308L45
but manifest.MediaType is already set from builtImage, and we built the image only if response header mediaType was known. 
but, i think i still can default to `SchemaTypes.DockerManifestV2` here for future proofing.

This can work for remote registry, but for archive tarball publishing, if oci image doesn't have mediaType it would mean that oci image tarball will have docker image.